torch/_dynamo/symbolic_convert.py

```diff
@@ -2444,3 +2444,45 @@ def YIELD_VALUE(self, inst: Instruction):
         self.generated_items.append(self.pop())
         # TODO(jansel): figure out why this is needed, it isn't in the docs for YIELD_VALUE
         self.push(ConstantVariable(None))
+
+    def GET_YIELD_FROM_ITER(self, inst):
+        tos = self.stack[-1]
+        if not isinstance(tos, ListIteratorVariable):
+            self.pop()
+            tos = ListIteratorVariable(
+                tos.items,
```

Comment:
Missing type checking before reading `tos.items` (which might not exist)

Reply:
sg

Reply:
not fixed?

Reply:
It might be easier to just do `BuildinVariable(iter).call_function(self, [tos], {})` since that function already does the needed error checking.

Reply:
Nice, that's a good point.